mgmtd/mgmt_vty.c

```diff
@@ -566,6 +567,9 @@ void mgmt_vty_init(void)
 	 * backend components that are moved to new MGMTD infra
 	 * here one by one.
 	 */
+	affinity_map_init();
+	if_cmd_init(NULL);
```

Comment:
`if_cmd_init` is already called in `mgmt_main.c`. Probably makes sense to move it here together with `vrf_cmd_init`, but we definitely don't want double initialization.